source/crud/tests/README.md

```diff
@@ -370,56 +370,15 @@ Assert that a CommandStartedEvent was observed for the `killCursors` command.
 
 ### 10. `MongoClient.bulkWrite` returns error for unacknowledged too-large insert
```

Comment:
Suggest removing the test name and simplifying to:

```
### 10. (Removed)
```

Reply:
If someone linked to the test directly, https://github.com/mongodb/specifications/blob/master/source/crud/tests/README.md#10-mongoclientbulkwrite-returns-error-for-unacknowledged-too-large-insert, the link won't work after such a change.

We could have started to use explicit [heading IDs, like in kramdown](https://kramdown.gettalong.org/syntax.html#specifying-a-header-id) to prevent similar issues in the future, but [GitHub Flavored Markdown](https://github.github.com/gfm/) supports them neither for [ATX headings](https://github.github.com/gfm/#atx-headings) nor for [setext headings](https://github.github.com/gfm/#setext-heading).

Reply:
Retracting suggestions.